src/components/navigation/Navigation.jsx

```diff
@@ -0,0 +1,37 @@
+import "./navigation.css";
+
+const Navigation = () => {
+  return (
+    <header id="header" className="header">
+      <div className="navbar-container">
```

Comment:
Suggestion: I think you can remove id attribute from here.
Suggestion: You can replace this <div> to <nav>